Index: src/main/resources/ui/ui2021/racunovodja/addZaposleni.fxml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/resources/ui/ui2021/racunovodja/addZaposleni.fxml	(date 1643217414643)
+++ src/main/resources/ui/ui2021/racunovodja/addZaposleni.fxml	(date 1643217414643)
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import java.lang.*?>
+<?import java.util.*?>
+<?import javafx.scene.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+
+<AnchorPane xmlns="http://javafx.com/javafx"
+            xmlns:fx="http://javafx.com/fxml"
+            fx:controller="ui.controller.AddZaposleniController"
+            prefHeight="400.0" prefWidth="600.0">
+
+</AnchorPane>
Index: src/main/java/ui/controller/AddZaposleniController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ui/controller/AddZaposleniController.java	(date 1643217421708)
+++ src/main/java/ui/controller/AddZaposleniController.java	(date 1643217421708)
@@ -0,0 +1,4 @@
+package ui.controller;
+
+public class AddZaposleniController {
+}
